.circleci/config.yml

```diff
@@ -38,8 +38,12 @@ jobs:
       - run:
           working_directory: ~/
           name: Set up swagger CLI
+          # This would be preferred, but would need to install ruby in the CircleCI docker image:
+          # cd workbench/ui && ruby -r ./libproject/devstart.rb -e download_swagger_codegen_cli
           command: |
-            cd workbench/tools && ./download_swagger_cli.sh
+            cd workbench/ui && wget https://storage.googleapis.com/swagger-codegen-cli/swagger-codegen-cli-2.3.0-20170814.101630-90.jar -O libproject/swagger-codegen-cli.jar
+          command: |
+            cd workbench/ui && ruby -r ./libproject/devstart.rb -e download_swagger_codegen_cli
```

Comment:
I think this is the same as the "this would be preferred" line, right? Obselete comment?